src/main/java/uk/gov/hmcts/divorce/ccd/NoFaultDivorce.java

```diff
@@ -20,7 +20,7 @@
 @Component
 public class NoFaultDivorce implements CCDConfig<CaseData, State, UserRole> {
 
-    public static final String CASE_TYPE = "NO_FAULT_DIVORCE17";
+    public static final String CASE_TYPE = "NO_FAULT_DIVORCE18";
```

Comment:
has the field type changed ?

Reply:
`screenHasMarriageBroken` has been renamed to: `applicant1ScreenHasMarriageBroken`. When I merged #359 in, it caused all existing cases on `NO_FAULT_DIVORCE17` to break when saving due to this field rename, so I had to revert it and create this PR to update the case type version.